cmd/kubeadm/app/preflight/checks.go

```diff
@@ -1061,32 +1052,34 @@ func RunChecks(checks []Checker, ww io.Writer, ignorePreflightErrors sets.String
 	found := []checkErrors{}
 
 	for _, c := range checks {
-		name := c.Name()
```

Comment:
i'd say this change is not needed. the intermediate variable has benefits, unless the compiler inlines all the `Name()` calls, probably yes, but we have no guarantee for that.

Reply:
yop